styles/styles.css

```diff
@@ -207,7 +207,6 @@ header {
 
 h1, h2, h3,
 h4, h5, h6 {
-  font-size: calc((var(--font-size) * 1px) - (var(--typography-sensitivity) * (var(--font-size) - var(--min-font-size)) * (var(--min-viewport-size) * 1px - 100vw)) / (var(--max-viewport-size) - var(--min-viewport-size)) - (var(--font-size) - var(--min-font-size)) * var(--typography-sensitivity) * 1px);
```

Comment:
please do not change this. This provides dynamic font size based on the screen width. If you want to override this default for a specific section, then please create a scoped rule for that.

Reply:
@ekremney strange, I don't remember having touch that at all.
But I did copy the styles.css from main as it was not in sync with my branch, let me review this